tests/common-utils.js

```diff
@@ -141,9 +141,7 @@ commonUtils.asyncLoadScript = function (url) {
       }
     };
 
-    // Attach the script tag to the page (before the first script) so the
-    //magic can happen.
-    firstScript.parentNode.insertBefore(script, firstScript);
+    document.body.append(script);
```

Comment:
good catch, for script async = true the order does not matter as per spec, and even if , the order would mostly depend on call order of asyncLoadScript, not tag order...